mleap-core/src/main/scala/ml/combust/mleap/core/feature/CountVectorizerModel.scala

```diff
@@ -34,8 +44,16 @@ case class CountVectorizerModel(vocabulary: Array[String],
     } else {
       termCounts.filter(_._2 >= effectiveMinTF).toSeq
     }
+    effectiveCounts
+  }
+
+  def apply(document: Seq[String]): Vector = {
+    Vectors.sparse(outputSize, _apply(document))
+  }
 
-    Vectors.sparse(dict.size, effectiveCounts)
+  def mleapApply(document: Seq[String]): Tensor[Double] = {
```

Comment:
I'm not sure I understand the naming. `apply` vs `mleapApply` ? 
One is the legacy Vectors.sparse `apply` and the other is the new one, should these be called with more informative names ?

Reply:
The old apply can just be dropped. Earlier i had incorrectly assumed that `CountVectorizerModel.apply`  is called by both mleap-runtime and spark. However, since this code is only called by  mleap runtime which sends expects mleap tensors, i can drop the legacy apply completely and just use the new one.

I can do the same change for Interaction

Reply:
I've updated the code to remove the legacy apply and just use `Tensor` instead of `Vector` in `InteractionModel` and `CountVectorizerModel`. We can do this since both of these transformers are being used only by the corresponding transformers `CountVectorizer` and `Interaction` in mleap runtime which expect `Tensor` outputs.